sslcert-review.py

```diff
@@ -0,0 +1,489 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""SSL Grader"""
+__author__ = "Kevin Amorin"
+__copyright__ = "Copyright 2020"
+__license__ = "GPL"
+__version__ = "1.0.1"
+
+import sys, os
+from shodan import Shodan
+from pprint import pprint, pformat
+import pickle
+from OpenSSL import crypto
+from datetime import datetime
+import certifi
+import pem
+import logging
+import argparse
+from prettytable import PrettyTable
+import csv
+import censys.certificates
+import censys.ipv4
+
+ROOT_STORE = None
+
+def log(s, type="DEBUG"):
+    """ log wrapper """
+    levels = {"DEBUG": 10, "INFO": 20, "WARNING": 30, "ERROR": 40, "CRITICAL": 50}
```

Comment:
runtime will be faster if you pull this dict out of this function and only define it once